WordPressComKit/RequestRouter.swift

```diff
@@ -0,0 +1,39 @@
+import Foundation
+import Alamofire
+
+public enum RequestRouter: URLRequestConvertible {
+    static let baseURLString = "https://public-api.wordpress.com/rest/v1.1/"
+    // FIXME: This needs to go somewhere better
+    public static var bearerToken = ""
```

Comment:
Yes. Each service instance should be able to have different OAuth2 tokens. See https://github.com/wordpress-mobile/WordPress-iOS/pull/4928/files#r55501013